env

```diff
@@ -0,0 +1,11 @@
+NODE_ENV=dev
```

Comment:
You were only supposed to update `example.env`. Why? because `REACT_APP_MAGIC_BOOST_SUPABASE_KEY` gives people complete access and putting that here, on a public repo (or any repo), leaks security information into the real world. 

Next steps: 
- Remove these files from the repository
- Revert the changes on `.gitignore`
- Re-generate the keys in Supabase